src/getMigrateDashboards.ts

```diff
@@ -42,7 +43,14 @@ export const getMigrateDashboards =
   ) =>
   <FromDashboardState, ToDashboardState>(
     callback: MigrateDashboardCallback<FromDashboardState, ToDashboardState>,
+    deserialize: (state: FromDashboardState) => FromDashboardState,
+    serialize: (state: ToDashboardState) => ToDashboardState,
```

Comment:
The input types are not accurate: theoretically it should be `From/ToDashboardState<serialized>`.
The input types are generic themselves so we cannot do that exactly, but we could use `unknown` (or `any` exceptionally if `unknown` causes too many issues) to avoid being mislading.